spec/interactors/find_user_by_reset_token_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe FindUserByResetToken do
+  describe ".call" do
+    let(:token) { Encryptor.generate_token }
+
+    context "when successful" do
+      before do
+        @user = create(:confirmed_user, reset_digest: token[0])
+      end
```

Comment:
Oh that darn rubocop! Always nagging :cop: 

Best practice says you tend to avoid using instance variables, and that's probably why rubocop is complaining. You can approach this in a couple different ways. 

One is to not use a variable at all and simply call `create` in the before block. You only have a single test that is using the variable to compare to the `interactor.user`. You could argue that you don't really need to validate that though. You can test that you are getting a user back but are not concerned that it was the user you created. You leave that to AR's `find_by` method.

You could also say: "but I really want to know that I'm getting the right guy". In that case you can make use or rspec's `let!` (let bang). Our known friend `let` works by lazy evaluating its block, in contrast `let!` evaluates before the example is executed.